### PR TITLE
Update doc cache tag

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: en/developer_guide/esi.rst [hunk 1]
```diff
@@ -85,6 +85,17 @@ The generic tags are added for all kind of blocks, they depend on :
 Additionally, each rendering strategy for blocks can define a list of specific tags
 by implementing the method ``DisplayBlockInterface::getCacheTags()``.
 
+By default, Open Orchestra provides some tag that can used in your rendering strategies :
```
Comment (veolys-nanne) [inline en/developer_guide/esi.rst]: can be used in the rendering strategies

File: en/developer_guide/esi.rst [hunk 2]
```diff
@@ -85,6 +85,17 @@ The generic tags are added for all kind of blocks, they depend on :
...
+
+* ``node- + NODE_ID`` : Invalidate when node status is updated.
+* ``contentType- + CONTENT_TYPE`` : Invalidate when content type is updated or deleted.
+* ``contentId- + CONTENT_ID`` : Invalidate when content status is updated or content is deleted.
+* ``mediaId- + MEDIA_ID`` : Invalidate when media is updated or deleted.
+* ``menu- + SITE_ID`` : Invalidate when a node of this site is deleted, restored, his status changed or his path is updated.
```
Comment (veolys-nanne) [inline en/developer_guide/esi.rst]: or his path updated

**Comment by alavieille** [comment]
@itkg-nanne pr updated

File: en/developer_guide/esi.rst [hunk 3]
```diff
@@ -85,6 +85,17 @@ The generic tags are added for all kind of blocks, they depend on :
...
+
+* ``node- + NODE_ID`` : Invalidate when node status is updated.
+* ``contentType- + CONTENT_TYPE`` : Invalidate when content type is updated or deleted.
+* ``contentId- + CONTENT_ID`` : Invalidate when content status is updated or content is deleted.
+* ``mediaId- + MEDIA_ID`` : Invalidate when media is updated or deleted.
+* ``menu- + SITE_ID`` : Invalidate when a node of this site is deleted, restored, his status changed or his path updated.
+
+These tags are gathered in the ``OpenOrchestra\BaseBundle\Manager\TagManager`` class.
+
+If this tags isn't enough,
```
Comment (veolys-nanne) [inline en/developer_guide/esi.rst]: It's possible to define new cache tags which would be flushed by calling
 `OpenOrchestra\DisplayBundle\Manager\CacheableManager::invalidateTags()`

File: en/developer_guide/esi.rst [hunk 4]
```diff
@@ -85,6 +85,17 @@ The generic tags are added for all kind of blocks, they depend on :
 Additionally, each rendering strategy for blocks can define a list of specific tags
 by implementing the method ``DisplayBlockInterface::getCacheTags()``.
 
+By default, Open Orchestra provides some tag that can be used in the rendering strategies:
```
Comment (ngilain) [inline en/developer_guide/esi.rst]: some tags

File: en/developer_guide/esi.rst (hunk 3 above)
Comment (ngilain) [inline en/developer_guide/esi.rst]: if these tags are not enough

**Comment by alavieille** [comment]
@itkg-ngilain @itkg-nanne pr updated

File: en/developer_guide/esi.rst [hunk 5]
```diff
@@ -85,8 +85,17 @@ The generic tags are added for all kind of blocks, they depend on :
 Additionally, each rendering strategy for blocks can define a list of specific tags
 by implementing the method ``DisplayBlockInterface::getCacheTags()``.
 
-It's possible to define new cache tags for some data to be flushed at optimal times,
-depending on the specifics of the application. These tags can then be flushed by calling
+By default, Open Orchestra provides some tags that can be used in the rendering strategies:
+
+* ``node- + NODE_ID`` : Invalidate when node status is updated.
```
Comment (ngilain) [inline en/developer_guide/esi.rst]: Automatically invalidated when ....

Same for the other tags

**Comment by ngilain** [comment]
:+1: with the lastmodifications

**Pull request merged**